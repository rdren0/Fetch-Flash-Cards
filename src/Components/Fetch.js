import React from 'react';

function Fetch() {
  return (
    <div className="fetch">
     <h1>You answered every card correctly!</h1>
     <br/>
     <iframe src="https://giphy.com/embed/xlYKItjhiDsY" width="480" height="270" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
    </div>
  );
}
  


export default Fetch;
