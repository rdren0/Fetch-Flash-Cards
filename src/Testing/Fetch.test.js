import Fetch from '../Components/Fetch';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });



describe('Fetch', () => {
  let wrapper;
  
  beforeEach(() =>{
    wrapper = shallow(
      <Fetch/>
    )
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});