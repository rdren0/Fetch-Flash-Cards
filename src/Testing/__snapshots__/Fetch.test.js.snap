// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fetch should match the snapshot with all data passed in correctly 1`] = `
<div
  className="fetch"
>
  <h1>
    You answered every card correctly!
  </h1>
  <br />
  <iframe
    allowFullScreen={true}
    class="giphy-embed"
    frameBorder="0"
    height="270"
    src="https://giphy.com/embed/xlYKItjhiDsY"
    title="Mean Girls Gif"
    width="480"
  />
</div>
`;
